### PR TITLE
Emily2

### PR DESCRIPTION
**Pull request by Emily484**
loginId and /home fix

### CODE REVIEW

File: client/src/components/Auth/Login.js [hunk 1]
```diff
@@ -25,24 +25,23 @@ class Login extends Component {
 
   onSubmit(e) {
     e.preventDefault();
-
     const user = {
-      loginId: this.state.loginId,
-      password: this.state.password
+      email: this.state.loginId,
```
Comment (slpriya) [inline client/src/components/Auth/Login.js]: Back-end we are using loginId

File: client/src/components/Auth/Login.js [hunk 2]
```diff
@@ -25,24 +25,23 @@ class Login extends Component {
...
+      password: this.state.password,
```
Comment (slpriya) [inline client/src/components/Auth/Login.js]: Remove comma

**Review by slpriya — APPROVED** [review]
Added a comments pls hava a look into it. Looks good

**Review by solongoch — APPROVED** [review]
Looks good

**Pull request merged**